File: src/app/shared/model/response/dashboardDTO.ts
import { VSaleDateDTO } from './vSaleDateDTO';
import { DailySaleDTO } from './dailySaleDTO';
import { MonthSaleDTO } from './monthSaleDTO';

export interface DashboardDTO{
    quantityProduct?:number;
    quantityCategory?:number;
    quantityMark?:number;
    quantityProvider?:number;
    quantityUnit?:number;
    quantityCustomer?:number;
    lstSaleDate:VSaleDateDTO[];
    lstDailySale:DailySaleDTO[];
    lstMonthSale:MonthSaleDTO[];
}

File: src/main/java/com/sys/botica/crce/pe/sys_botica/dto/DashboardDTO.java
package com.sys.botica.crce.pe.sys_botica.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DashboardDTO {
	private Integer quantityProduct;
	private Integer quantityCategory;
	private Integer quantityMark;
	private Integer quantityProvider;
	private Integer quantityUnit;
	private Integer quantityCustomer;
	private List<VSaleDateDTO> lstSaleDate;
	private List<DailySaleDTO> lstDailySale;
	private List<MonthSaleDTO> lstMonthSale;
}
